docs/hugo/content/contributing/create-a-new-release.md

```diff
@@ -12,7 +12,7 @@ title: Creating a new release
 8. Ensure that the action associated with your release finishes successfully.
 9. Create a new PR to update our documentation and move resources listed under "Next Release" to the heading "Released" by checking out `main`, running `task` and creating a PR of the results.
```

Comment:
Should this entry link to the new section you added below (#update-documentation)?

Reply:
Yes. Done.